nautobot/dcim/tests/test_views.py

```diff
@@ -2362,6 +2362,12 @@ class ConsoleConnectionsTestCase(ViewTestCases.ListObjectsViewTestCase):
     def _get_base_url(self):
         return "dcim:console_connections_{}"
 
+    def get_list_url(self):
+        return "/dcim/console-connections/"
+
+    def get_title(self):
+        return "Console Connections"
```

Comment:
Why were you required to hard-code these for these three test cases that facilitated the need for abstract methods as well?

Reply:
For `ConsoleConnections`, `PowerConnections` and `InterfaceConnections`. `list_url` and `title` is overriden in `extra_context()` https://github.com/nautobot/nautobot/blob/c9345f0ae1e9a1765a3212c276d739971ba2d0dd/nautobot/dcim/views.py#L2776
whereas all the other models are using ObjectListView's context https://github.com/nautobot/nautobot/blob/c9345f0ae1e9a1765a3212c276d739971ba2d0dd/nautobot/core/views/generic.py#L341.